index.rst

```diff
@@ -7,235 +7,260 @@
+To store in this area requires a Zephyr Jira Test Case counterpart.
 Details on how to use this area is found in the README file of the repository.
 
 Review Criteria
 """""""""""""""
 
-This section will be populated in future versions.
-However, as mentioned previously, the expectation is that the notebooks closely resemble the examples found in the `AuxTel area <https://github.com/lsst-ts/ts_notebooks/tree/develop/procedures/auxtel>`_.
+Official artifacts for verification require specified review criteria.
+Standardizing this is underway, and this section will be populated in a future revision of this document.
+As mentioned previously, the expectation is that the notebooks closely resemble the `AuxTel area in lsst-ts/ts_notebook <https://github.com/lsst-ts/ts_notebooks/tree/develop/procedures/auxtel>`_ example.
 
 Webspaces
 ^^^^^^^^^
 
-During early operations of the Auxiliary Telescope, there was a need to have the information required for operators assembled into a single area with a coherent, searchable structure.
-Following from the examples used by DM, specifically the architecture used for `DM Pipelines <https://pipelines.lsst.io>`_ documentation, two webspaces have been created to support this purpose. 
-These are described in the subsections below.
+During early operations of the Auxiliary Telescope, there was a need to have the information required for operators be assembled into a single area with a coherent, searchable structure.
+The architecture, sometimes referred to as *user guides* or *webspaces*, for `The LSST Science Pipelines documentation <https://pipelines.lsst.io>`_ was extensively used and readily available.
+To an end user, webspaces are very similar to technotes -- with the name space of the requstor's choice, it will be created with an associated GitHub repository and website with the name space, with the URL form ``namespace.lsst.io``.
+Two webspaces have been created to support efforts with the Auxiliary Telescope, and they are described in the subsections below.
+
+.. note::
+
+   Adding a ``/v`` to a webspace URL will display the rendered versions of all branches in the repository.
 
 Webspaces are areas are best used for user-facing documentation.
-This includes general information, how-to's and procedures that are not subject to change control, and therefore do not risk the safety of personnel or equipment.
-Anything requiring strict reviews (e.g. glass lift plans) cannot be put into a webspace, but they can be linked!
+This includes general information, how-to documentation and procedures that are not subject to change control (therefore, these procedures are restricted to ones that do not risk the safety of personnel or equipment).
+Anything requiring strict reviews (e.g., glass lift plans) cannot be put into a webspace, but they can be linked within webspaces.
```

Comment:
"from webpaces" no?
They will be hosted elsewhere (e.g docushare)

Reply:
Changed in latest commit to better indicate that this type of information is never in a webspace.